### PR TITLE
Refactor buttons

### PR DESCRIPTION
**Pull request by wrightmartin**

### CODE REVIEW

File: assets/stylesheets/responsive/custom.scss [hunk 1]
```diff
@@ -604,6 +638,14 @@ a.link_to_this {
   }
 }
 
+.feed_link_sidebar .link_button_green {
+  @include button-secondary();
```
Comment (garethrees) [inline assets/stylesheets/responsive/custom.scss]: I think the follow link should be a primary action

**Comment by garethrees** [comment]
Report button has blue text, follow has black:

screen shot 2015-08-21 at 11 08 13

File: assets/stylesheets/responsive/custom.scss [hunk 2]
```diff
@@ -121,57 +121,91 @@ input.use-datepicker[type=text] {
...
+@mixin button-secondary($args...) {
+  @include button-base($args...);
+    background-color: desaturate(darken($color_sand, 10%), 5%);
+    color: $color_black;
+    font-weight: normal;
+    &:hover,
+    &:active,
+    &:focus {
+      color: $color_black;
+      background-color:  desaturate(darken($color_sand, 15%), 5%);
```
Comment (garethrees) [inline assets/stylesheets/responsive/custom.scss]: I still find the hover contrast too low on these. 25% seemed better.

**Comment by wrightmartin** [comment]
The 'back to results' link isn't a button - it's a ```.message```

**Pull request merged**